### DISCUSSION TITLE
Date not localized in multi-site config

### DISCUSSION THREAD

**Discussion by Sennik** [statamic/cms#6916] [Troubleshooting] [answered]
I have a multi-site with the same site in different locales:

```
   'sites' => [
        'default' => [
            'name' => 'NL Website',
            'locale' => 'nl_NL',
            'url' => 'https://nl.website.com/',
        ],
        'english' => [
            'name' => 'EN Website',
            'locale' => 'en_US',
            'url' => 'https://uk.website.com/',
        ],
        ...
   ]
```

When I output a simple date {{ date }} in my antlers template the date is always in English. In the template it shows the right locale (<html lang="{{ site:short_locale }}"> = <html lang="nl">). If I set locale to 'nl' it is also still in English. Am I forgetting something?

**Comment by jasonvarga** [comment, marked as answer]
Don't use `{{ date }}` by itself. Use `{{ date iso_format="" }}` with the appropriate format.

https://carbon.nesbot.com/docs/#iso-format-available-replacements

**Comment by Sennik** [comment]
I know, but what when I use this template for multiple locales? Wouldn't it be nice to create date/numbers formats in de sites.php configuration so that the formatting is covered already?

Likes this:

```
   'sites' => [
        'default' => [
            'name' => 'NL Website',
            'locale' => 'nl_NL',
            'url' => 'https://nl.website.com/',
            'date-format' => '',
            'numbers-format' => '',
            'currency-format' => '',
        ],
        'english' => [
            'name' => 'EN Website',
            'locale' => 'en_US',
            'url' => 'https://uk.website.com/',
            'date-format' => '',
            'numbers-format' => '',
            'currency-format' => '',
        ],
        ...
   ]
```

**Reply by jasonvarga** [reply]
You can add additional site attributes, then use that in the `iso_format` modifier.

https://statamic.dev/multi-site#additional-attributes

**Reply by Sennik** [reply]
Aaaaah, great, of course! Thanks!
